modules/dnn/src/model.cpp

```diff
@@ -20,23 +20,26 @@ namespace dnn {
 struct Model::Impl {
     Size   size;
     Scalar mean;
-    float  scale;
-    bool   swapRB;
-    bool   crop;
+    float  scale = 1.0;
+    bool   swapRB = true;
```

Comment:
By default, preprocessing all the preprocessing stages should be disabled. So please turn `swapRB` to false by default.